### DISCUSSION TITLE
Pagination@UI level

### DISCUSSION THREAD

**Discussion by 1988gadocansey** [jasontaylordev/CleanArchitecture#512] [Q&A]
How do I implement the pagination with the Angular Component
<img width="564" alt="Screen Shot 2021-11-26 at 11 43 34 AM" src="https://user-images.githubusercontent.com/7513799/143575951-bdc8b4e7-575d-4423-a39e-10e7f10c3275.png">

Fetched data from  backend using
<img width="1055" alt="Screen Shot 2021-11-26 at 11 44 41 AM" src="https://user-images.githubusercontent.com/7513799/143576076-c3ea49a9-c5dd-4e6a-8366-923471b3ded1.png">
g

**Comment by suicodave** [comment]
Have you tried binding totalItems with TotalCount from the api response?
```html
<pagination [totalItems]="totalCount"><pagination>
```

**Reply by 1988gadocansey** [reply]
I will check if  TotalCount  is available in the api response and revert back.
